maintenance/ebpf/enabling-bpf.md

```diff
@@ -66,8 +67,9 @@ eBPF mode has the following pre-requisites:
   If you must use an overlay, we recommend that you use VXLAN, not IPIP.  This is because IPIP performs poorly with eBPF mode due 
   to technical limitations in the kernel.
   
-- The underlying network must be configured to allow VXLAN packets between {{site.prodname}} hosts.  In eBPF mode, 
-  VXLAN is used to forward traffic to Kubernetes NodePorts, while preserving source IP.
+- The underlying network must be configured to allow VXLAN packets between {{site.prodname}} hosts (even if you normally 
+  use IPIP or non-overlay for Calico traffic).  In eBPF mode, VXLAN is used to forward traffic to Kubernetes NodePorts, 
+  while preserving source IP.  eBPF mode honours the Felix `VXLANMTU` setting (see [Configuring MTU]({{ site.baseurl }}/networking/mtu)).
```

Comment:
Do you want to say "so make sure it is configured correctly for your environment"?

Reply:
Oh hang on - this will be going in after auto-MTU, so it is likely less relevant to manually set it.